Pod/Classes/BONTag.h

```diff
@@ -0,0 +1,69 @@
+//
+//  BONTag.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "BONCompatibility.h"
+
+@protocol BONTextable;
+
+@interface BONTag : NSObject <NSCopying>
+
+@property (copy, nonatomic, readonly, BONNonnull) NSString *startTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *endTag;
+@property (copy, nonatomic, readonly, BONNonnull) NSString *escapeString;
+@property (copy, nonatomic, readonly, BONNonnull) id<BONTextable> textable;
+
+- (BONNonnull instancetype)initWithStartTag:(BONNonnull NSString *)startTag endTag:(BONNonnull NSString *)endTag escapeString:(BONNonnull NSString *)escapeString textable:(BONNonnull id<BONTextable>)textable;
```

Comment:
…but here's an idea, @Imperiopolis. You could mark the convenience functions for `BONTagComplexMake` as `NS_SWIFT_UNAVAILABLE`, and add a few more `-initWith...` methods to cover the cases where you would normally use default arguments. You would just never call those in Obj-C code because you would use the convenience functions. Then, the only way to initialize `BONTag` objects in Swift would be to use one of those initializers from Obj-C.

Since it would be a breaking change for Swift users if we marked the convenience functions as unavailable, I'd prefer to do it now. @Imperiopolis, let me know if you'd like to do it, and if not, I'll take care of it.